content/docs/components/misc/metadata.md

```diff
@@ -178,8 +178,10 @@ curl -X POST http://localhost:8080/api/v1alpha1/artifact_types \
 
 ## Next steps
 
-Run the 
+- Run the 
 [xgboost-synthetic notebook](https://github.com/kubeflow/examples/tree/master/xgboost_synthetic)
 to build, train, and deploy an XGBoost model using Kubeflow Fairing and Kubeflow
 Pipelines with synthetic data. Examine the metadata output after running
 through the steps in the notebook.
+
+- Besides using the Python SDK to log metadata directly, you can add your own [metadata watcher](https://github.com/kubeflow/metadata/blob/master/watcher/README.md) to watch Kubernetes resource changes and save the metadata into the metadata service.
```

Comment:
Thanks @zhenghuiwang This looks very good. Would it make sense to move this section up higher in the doc? For example, it could fit between **Learn more about the Metadata SDK** and **Tracking artifacts on the Metadata UI**. That way, people would see immediately that the watcher is an alternative to the SDK, and they'd understand that the metadata recorded by the watcher is also visible on the UI.

Reply:
Good suggestion! I made the change. Thanks.